test/uk/gov/hmrc/agentservicesaccount/controllers/AgentServicesControllerSpec.scala

```diff
@@ -89,6 +88,9 @@ class AgentServicesControllerSpec extends WordSpec with Matchers with OptionValu
       content should not include "Agent Services Account"
       content should include(messagesApi("agent.services.account.heading"))
       content should include(messagesApi("agent.services.account.heading.summary"))
+      content should include(messagesApi("agent.services.account.additional.links.title"))
+      content should include(messagesApi("agent.services.account.additional.links.mapping.body"))
+      content should include(messagesApi("agent.services.account.additional.links.mapping.url"))
       content should include(arn)
       content should include(signOutUrl)
```

Comment:
Added your feedback to: https://github.com/hmrc/agent-services-account-frontend/pull/16